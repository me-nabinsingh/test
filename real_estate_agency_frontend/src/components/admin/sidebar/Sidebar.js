import React from 'react';
import { Link } from 'react-router-dom';

import './styles.css';


const Sidebar = () => {
    return ( 
        <nav className="col-md-2 d-none d-md-block bg-light sidebar">
          <div className="sidebar-sticky  mt-5">
            <ul className="nav flex-column py-2">
              <li className="nav-item">
                <Link className="nav-link" to="/admin/">
                  Dashboard 
                </Link>
              </li>
              <li className="nav-item">
                <Link className="nav-link" to="/admin/banners">
                  Banners
                </Link>
              </li>
              <li className="nav-item">
                <Link className="nav-link" to="/admin/planets">
                  Planets
                </Link>
              </li>
              <li className="nav-item">
                <Link className="nav-link" to="/admin/images">
                  Images
                </Link>
              </li>
            </ul>

          </div>
        </nav>
     );
}
 
export default Sidebar;