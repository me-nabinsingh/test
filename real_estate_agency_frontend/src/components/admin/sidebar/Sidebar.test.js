import React from 'react';
import Sidebar from './Sidebar';
import {Router } from "react-router-dom";
import renderer from 'react-test-renderer';
import history from "../../../util/history";

describe('<Sidebar />', () => {

    it('renders without crashing', () => {
        const component = renderer.create(
            <Router history={history}>
                <Sidebar/>
            </Router>
        );

        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});