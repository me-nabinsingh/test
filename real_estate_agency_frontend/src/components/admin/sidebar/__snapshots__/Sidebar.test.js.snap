// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Sidebar /> renders without crashing 1`] = `
<nav
  className="col-md-2 d-none d-md-block bg-light sidebar"
>
  <div
    className="sidebar-sticky  mt-5"
  >
    <ul
      className="nav flex-column py-2"
    >
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/admin/"
          onClick={[Function]}
        >
          Dashboard
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/admin/banners"
          onClick={[Function]}
        >
          Banners
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/admin/planets"
          onClick={[Function]}
        >
          Planets
        </a>
      </li>
      <li
        className="nav-item"
      >
        <a
          className="nav-link"
          href="/admin/images"
          onClick={[Function]}
        >
          Images
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
